dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -3415,11 +3420,13 @@ RtpsUdpDataLink::RtpsWriter::gather_nack_replies_i(MetaSubmessageVec& meta_subme
       RepoId destination;
       if (proxy.contains(seq, destination)) {
         if (destination == GUID_UNKNOWN) {
-          consolidated_recipients.insert(addrs.begin(), addrs.end());
```

Comment:
Why is there no update to the recipients here?

Reply:
ditto above